packages/cli-kit/src/public/node/output.ts

```diff
@@ -306,20 +306,6 @@ export function outputWarn(content: OutputMessage, logger: Logger = consoleWarn)
   outputWhereAppropriate('warn', logger, message)
 }
 
-/**
- * Outputs a warning message to the user with some error styling.
- * Warning messages receive a special formatting to make them stand out in the console.
- * Note: Warning messages are sent through the standard output.
- *
- * @param content - The content to be output to the user.
- * @param logger - The logging function to use to output to the user.
- */
-export function outputWarnError(content: OutputMessage, logger: Logger = consoleLog): void {
```

Comment:
Why removing this?

Reply:
It's an output utility method added to display an stylish error message when the `dev preview mode` operation fails but the execution of the command continues. 
Now it's unused so I think it's better to remove all dead code if possible